### ISSUE TITLE
error

### ISSUE THREAD

**Issue by Disqu3-mirOir-qt** [TeamNewPipe/NewPipe#7125]
## Exception
* __User Action:__ ui error
* __Request:__ ACRA report
* __Content Country:__ FR
* __Content Language:__ fr
* __App Language:__ fr
* __Service:__ none
* __Version:__ 0.21.10
* __OS:__ Linux Android 8.0.0 - 26
<details><summary><b>Crash log </b></summary><p>

```
io.reactivex.rxjava3.exceptions.OnErrorNotImplementedException: The exception was not handled due to missing onError handler in the subscribe() method call. Further reading: https://github.com/ReactiveX/RxJava/wiki/Error-Handling | java.lang.NullPointerException: Attempt to read from field 'androidx.recyclerview.widget.RecyclerView org.schabi.newpipe.databinding.FragmentSearchBinding.suggestionsList' on a null object reference
	at io.reactivex.rxjava3.internal.functions.Functions$OnErrorMissingConsumer.accept(Functions.java:717)
	at io.reactivex.rxjava3.internal.functions.Functions$OnErrorMissingConsumer.accept(Functions.java:714)
	at io.reactivex.rxjava3.internal.observers.LambdaObserver.onError(LambdaObserver.java:77)
	at io.reactivex.rxjava3.internal.observers.LambdaObserver.onNext(LambdaObserver.java:67)
	at io.reactivex.rxjava3.internal.operators.observable.ObservableObserveOn$ObserveOnObserver.drainNormal(ObservableObserveOn.java:201)
	at io.reactivex.rxjava3.internal.operators.observable.ObservableObserveOn$ObserveOnObserver.run(ObservableObserveOn.java:255)
	at io.reactivex.rxjava3.android.schedulers.HandlerScheduler$ScheduledRunnable.run(HandlerScheduler.java:123)
	at android.os.Handler.handleCallback(Handler.java:808)
	at android.os.Handler.dispatchMessage(Handler.java:101)
	at android.os.Looper.loop(Looper.java:166)
	at android.app.ActivityThread.main(ActivityThread.java:7529)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.Zygote$MethodAndArgsCaller.run(Zygote.java:245)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:921)
Caused by: java.lang.NullPointerException: Attempt to read from field 'androidx.recyclerview.widget.RecyclerView org.schabi.newpipe.databinding.FragmentSearchBinding.suggestionsList' on a null object reference
	at org.schabi.newpipe.fragments.list.search.SearchFragment.handleSuggestions(SearchFragment.java:970)
	at org.schabi.newpipe.fragments.list.search.SearchFragment.lambda$initSuggestionObserver$13$SearchFragment(SearchFragment.java:813)
	at org.schabi.newpipe.fragments.list.search.-$$Lambda$SearchFragment$0KRipyJJS1-v4wlqLTlO9OsWksw.accept(Unknown Source:4)
	at io.reactivex.rxjava3.internal.observers.LambdaObserver.onNext(LambdaObserver.java:63)
	... 10 more

```
</details>
<hr>

**Comment by TobiGr** [comment]
Which RC was that? It should be fixed in the latest RC

**Comment by litetex** [comment]
Looks similar to https://github.com/TeamNewPipe/NewPipe/issues/6976

**Comment by TobiGr** [comment]
Correct. I fixed a part of this exception in #7056. So I need to know which 0.21.10 RC was used to ensure the crash is gone / still present.
